.github/workflows/call-plugin-workflow.yml

```diff
@@ -0,0 +1,36 @@
+name: Run plugin workflow
+
+on:
+    pull_request:
+    workflow_dispatch:
+    push:
+        branches:
+            - master
```

Comment:
would you mind changing this already to "main"? so we do not need to do another PR to change it, once the repo is moved